toolbox.py

```diff
@@ -396,3 +405,55 @@ def identifyObtrusiveness(blockStr):
     if blockStr in lookup.OBTRUSIVE:
         return 3
     return 4
+
+
+def visualizeHeightmap(*arrays, title=None, autonormalize=True):
+    """**Visualizes one or multiple numpy arrays**.
+
+    Args:
+        title (str, optional): display title. Defaults to None.
+        autonormalize (bool, optional): Normalizes the array to be
+            between 0 (black) and 255 (white). Defaults to True.
```

Comment:
Perhaps the `Args` section should be removed and revisited when `auto-documentation` is implemented?

Reply:
Hm yeah, I'd be ok with it for the sake of making it consistent. Long term, if we want a real documentation documenting arguments should be a part of it though.
This comes from the vscode python extension by the way, which generates it like this, I'm not sure which standard it follows:
```
    """[summary]

    Args:
        title ([type], optional): [description]. Defaults to None.
        autonormalize (bool, optional): [description]. Defaults to True.
    """
```